ipn/ipnlocal/local.go

```diff
@@ -3141,37 +3127,45 @@ func (b *LocalBackend) peerAPIServicesLocked() (ret []tailcfg.Service) {
 // painstakingly constructing it in twelvety other places.
```

Comment:
time to update this comment? maybe even remove this TODO now?

Reply:
Removed in the other pr